features/step_definitions/basics_steps.rb

```diff
@@ -0,0 +1,20 @@
+When("I visit the {string}") do |string|
+  visit root_path
+end
+
+Then("I should see {string}") do |text|
```

Comment:
In order for consistency across PRs, maybe we can change |text| to |content|?

Reply:
changed!